### PR TITLE
NR-172722 added docs for manually installing the agent

### PR DESCRIPTION
**Pull request by mbruin-NR**
This pr adds documentation for manually installing the iOS agent by downloading it and adding the xcframework to your Xcode project.

### CODE REVIEW

**Review by cdillard-NewRelic — APPROVED** [review]
LGTM 👍

File: src/nav/mobile-monitoring.yml [hunk 1]
```diff
@@ -57,6 +57,8 @@ pages:
                     path: /docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation
                   - title: Swift package manager
                     path: /docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation
+                  - title: Manually copy XCFramework
```
Comment (ally-sassman) [inline src/nav/mobile-monitoring.yml:60]:
```suggestion
                  - title: XCFramework manual installation
```

File: src/nav/mobile-monitoring.yml [hunk 2]
```diff
@@ -57,6 +57,8 @@ pages:
...
+                    path: /docs/mobile-monitoring/new-relic-mobile-ios/installation/manual-install
```
Comment (ally-sassman) [inline src/nav/mobile-monitoring.yml:61]:
```suggestion
                    path: /docs/mobile-monitoring/new-relic-mobile-ios/installation/manual-installation
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </Step>
+    <Step>In your Xcode project, add the unzipped agent:
+
+1. Right-click on your Project navigator and click **Add Files to `[YourProjectName]`**.
+2. Select the `NewRelic.xcframework` directory and click **Add**.        
+    </Step>
+    <Step>
+Embed the iOS agent:
+1. In the Project navigator, open your project and target settings.
+2. Click on the target to which you want to add the agent.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:26]: @mbruin-NR How do they know/decide which target to add the agent? I think we should add more on this, like if we have recommendations on where to add it

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+---
+title: XCFramework manual installation
+metaDescription: How to instrument to your iOS applications by downloading the XCFramework.
+freshnessValidatedDate: 2024-7-16
+---
+
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic iOS agent by downloading the XCFramework.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:7]:
```suggestion
We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic iOS agent (in an XCFramework package) and add it to your XCode project. 
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: XCFramework manual installation
+metaDescription: How to instrument to your iOS applications by downloading the XCFramework.
+freshnessValidatedDate: 2024-7-16
+---
+
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic iOS agent (in an XCFramework package) and add it to your XCode project. 
+
+## Manually add the New Relic XCFramework to your project
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:9]:
```suggestion
## Manually add the New Relic XCFramework to your project [#manual-install]
```

Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:9]:
```suggestion
```

Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:9]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic iOS agent (in an XCFramework package) and add it to your XCode project. 
+
+<Steps>
+    <Step>
+Download the latest iOS agent (XCFramework) from our [iOS agent release notes](/docs/release-notes/mobile-release-notes/ios-release-notes).        
+    </Step>
+    <Step>
+Navigate to where the iOS agent was downloaded and unzip it.        
+    </Step>
+    <Step>In your Xcode project, add the unzipped agent:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:16]:
```suggestion
    <Step>In your Xcode project, add the unzipped XCFramework:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx (hunk 3 above)
Comment (mbruin-NR) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx:24]: _Targets_ is a key word in Xcode development. It is the umbrella under which your app is constructed in you project. If the reader has used Xcode to make an iOS, tvOS, watchOS project I think they will understand the sentence.

**Review by ally-sassman — APPROVED** [review]

**Pull request merged**